### PR TITLE
Add Postcss to web-application template ;; WLFE-76

### PR DESCRIPTION
**Pull request by ofrogon**
Also:
- Update dependencies

### CODE REVIEW

File: templates/web-application/package.json [hunk 1]
```diff
@@ -25,48 +25,51 @@
...
+        "eslint": "8.43.0",
         "html-webpack-plugin": "5.5.3",
         "jest": "29.5.0",
         "jest-environment-jsdom": "29.5.0",
         "jest-junit": "16.0.0",
         "lodash-es": "4.17.21",
         "msw": "1.2.2",
-        "storybook": "7.0.20",
+        "postcss": "8.4.24",
+        "postcss-load-config": "4.0.1",
```
Comment (alexasselin008) [inline templates/web-application/package.json:63]: i'm not sure what the postcss-load-config is used for. it's the first time i see it. Pat made a sample app in the wl-config repo and it doesn't use it, so i think we could remove it. 

https://github.com/workleap/wl-web-configs/blob/main/sample/app/package.json

File: templates/web-application/postcss.config.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('postcss-load-config').Config} */
```
Comment (alexasselin008) [inline templates/web-application/postcss.config.js:1]: i would expect /** @type {import("postcss").Postcss} */

instead of the load-config import

**Review by alexasselin008 — APPROVED** [review]

File: templates/web-application/postcss.config.js (hunk 2 above)
Comment (ofrogon) [reply templates/web-application/postcss.config.js:1]: It doesn't work with `/** @type {import("postcss").Postcss} */`. This import is based on PostCSS documentation https://github.com/postcss/postcss#css-in-js

File: templates/web-application/package.json (hunk 1 above)
Comment (ofrogon) [reply templates/web-application/package.json:63]: This is required if I want to type correctly the `postcss.config.js`.

The other option is to ignore the `@type` specification in `postcss.config.js`.

File: templates/web-application/postcss.config.js (hunk 2 above)
Comment (alexasselin008) [reply templates/web-application/postcss.config.js:1]: i would remove the JSDoc typings then. It's not worth importing the package

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**